flink-table/flink-sql-client/src/test/resources/sql/set.q

```diff
@@ -42,20 +42,11 @@ CREATE TABLE hive_table (
 [INFO] Table has been created.
 !info
 
-# list the configured configuration
-set;
-table.sql-dialect=hive
-!ok
-
```

Comment:
Currently the set command will print all the commands in the configuration, e.g. 

```
execution.attached=true
execution.runtime-mode=batch
execution.savepoint.ignore-unclaimed-state=false
execution.shutdown-on-attached-exit=false
execution.target=remote
jobmanager.rpc.address=localhost
pipeline.classpaths=
pipeline.jars=file:/var/folders/1k/kcnx0zh91c1dkl0_m00ds2mh0000gp/T/junit3565817602485898043/test-jar/test-classloader-udf.jar
rest.port=63196
sql-client.execution.result-mode=tableau
[DEPRECATED] execution.result-mode=tableau
[DEPRECATED] execution.type=batch
```

In the test, we need to use jar the location of the jar is random. Therefore, I remove the test about the set.

Reply:
I think we must resolve this problem. Otherwise, we can't never test `SET` command. I have some ideas:

1) support dynamic variable in the `*.q` script, e.g. `$VAR_PIPELINE_JARS`, and replace the varibles in `CliClientITCase` after read script file. Just like what we do for the yaml files. 
2) support `SET LIKE 'table.*'` syntax to just show some specific configurations. But this need another issue to discuss and still can't test `SET` command. But I think this can be a useful feature if the configuration list is very long. 
3)  If we can use `StreamExecutionEnvironment.getExecutionEnvironment()` for initialization instead of constuctor, we may not need the MiniCluster configuration as the default configuration.